dart/common/NameManager.h

```diff
@@ -189,6 +227,12 @@ class NameManager
     return (mMap.find(_name) != mMap.end());
   }
 
+  /// Return true if the object is contained
+  bool hasObject(T _obj) const
+  {
+    return (mReverseMap.find(_obj) != mReverseMap.end());
+  }
+
   /// Get the number of the objects currently stored by the NameManager
   size_t getCount() const
   {
```

Comment:
Right, but a non-virtual function still can be hidden by the derived class.

getCount() is for returning the size so comparing the sized would be enough.